recipes/ipart/meta.yaml

```diff
@@ -0,0 +1,49 @@
+package:
+  name: ipart
+  version: 2.0.1
+
+source:
+  url: https://github.com/ihesp/IPART/archive/v2.0.1.tar.gz
+  sha256: f07275dd94568cc0da2d9f6b6582068682716124a72410182ca2b37692bc6678
+
+build:
+  noarch: python
+  number: 0
```

Comment:
Replace `build.sh` and `bld.bat` with `script` line:

https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L35